resource-definitions/template-driver/annotations/annotations.yaml

```
@@ -13,11 +13,30 @@ entity:
       templates:
         outputs: |
           update:
+            - op: add
+              path: /spec/deployment/annotations
+              value:
+                {{- range $key, $val := .resource.spec.deployment.annotations }}
+                {{ $key }}: {{ $val | quote }}
+                {{- end }}
+            - op: add
+              path: /spec/pod/annotations
+              value:
+                {{- range $key, $val := .resource.spec.pod.annotations }}
+                {{ $key }}: {{ $val | quote }}
+                {{- end }}
+            # If the Score file also defines a service, add annotations to the service object
             {{- if .resource.spec.service }}
-            {{- $port := values .resource.spec.service.ports | first }}
+            - op: add
+              path: /spec/service/annotations
+              value:
+                {{- range $key, $val := .resource.spec.service.annotations }}
+                {{ $key }}: {{ $val | quote }}
+                {{- end }}
+                {{- $port := values .resource.spec.service.ports | first }}

```

I think this line should be de-indented a bit, or moved to just above the `value:` line in the next `add` block

```suggestion
            {{- $port := values .resource.spec.service.ports | first }}

```